### PR TITLE
Add linting workflow

### PR DESCRIPTION
**Pull request by davbrito**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-</head>
-
-<body>
-  <x-app></x-app>
-</body>
-
-</html>
\ No newline at end of file
+  <body>
+    <x-app></x-app>
```
Comment (JuanVqz) [inline src/index.html:11]: is this a standard? I mean using the  `x-app` component.

**Review by JuanVqz — COMMENTED** [review]
cool! Gracias @davbrito, esto urge!

**Review by JuanVqz — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (davbrito) [reply src/index.html:11]: No. It's just our main web component where all the app lives.

Comment (JuanVqz) [reply src/index.html:11]: I have seen React and Vue mostly use the `#app` id, to mount the component `App`.

Comment (davbrito) [reply src/index.html:11]: With Web Components you use the component tag as a regular HTML element. No need to have an insertion point element.

Comment (JuanVqz) [reply src/index.html:11]: Got it

**Pull request merged**